layouts/partials/button-language.html

```diff
@@ -1,5 +1,11 @@
-{{ if .IsTranslated }}
-    {{ range .Translations }}
-        <a href="{{ .Permalink }}" class="lang">{{ .Site.Language.Lang }}</a>
-    {{ end}}
-{{ end }}
+<ul>
+  <li><a href="{{ "countries" | relLangURL }}">{{ $.Site.Params.countries }}</a></li>
+  <li><a href="{{ "tracking" | relLangURL }}">{{ $.Site.Params.analytics }}</a></li>
```

Comment:
Why put traking in the header ?

Reply:
Because I thought it would be cool to have 2 items in the nav bar, but it doesn't make any sense...